### DISCUSSION TITLE
Best way to upgrade an old station_logbooks branch installation

### DISCUSSION THREAD

**Discussion by nolith** [magicbug/Cloudlog#2313] [Q&A]
Hello, I've been away from hamradio activities since last October.

I am running my own cloud log server from the station_logbooks branch 6a401dfaebecde9d717a7b4ff42385cad5c09f17
It's quite old and from the project release pages seems that a lot happened in those months.

What will be the best way to upgrade my installation (it's installed from git as I was contributing as well back then)

Thank you
IU5BON

**Comment by magicbug** [comment]
I'd suggest take a back up and then switch branch to master if you was on the logbooks branch i'd hope it would be safe to do the jump
